src/styles/main.scss

```diff
@@ -1,3 +1,60 @@
 body {
   margin: 0;
 }
+
+$cell-size: 100px;
+$border-width: 1px;
```

Comment:
you use it once, it's not that kind of value which needed to be stored in a variable